jsbank/category-manage.js

```diff
@@ -0,0 +1,70 @@
+import {HISTORY_CATEGORY} from './initial.js';
+
+const listIn = document.getElementById("list_in");
+const listOut = document.getElementById("list_out");
+
+/*최초 렌더링*/
+HISTORY_CATEGORY.forEach((element)=>{
+    addCategory(element);
+});
+
+/*카테고리 화면에 추가*/
+function addCategory(element){
+    const list = document.createElement("li");
+    
+    list.innerHTML = `
+    <span>${element.name}</span>
+    `;  
+
+    element.type === "input"
+    ? listIn.appendChild(list)
+    : listOut.appendChild(list);
+}
+
+/*새 카테고리 생성*/
+const inCategory = document.getElementById("categorypg_input");
+const outCategory = document.getElementById("categorypg_output");
+
+function createCateogry(element){
+    if(element.keyCode === 13){
```

Comment:
```suggestion
function createCateogry(event){
    if(event.keyCode === 13){
```
여기는 event 객체를 받아오는거니까, e나 event로 변수명을 지어줘야 event객체를 사용하고 있구나 한 눈에 알아보기 쉽겠죠 ?!